scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -119,6 +110,32 @@ object AcceptAllPathFilter extends PathFilter {
 }
 
 object FileSource {
+  val LOG = LoggerFactory.getLogger(this.getClass)
+
+  val VerboseFileSourceLoggingKey = "com.twitter.scalding.filesource.verbose.logging"
```

Comment:
maybe add a short line describing the use of this param? Add if we have a sense of the overhead it causes, that as well?